### ISSUE TITLE
support for opset11

### ISSUE THREAD

**Issue by guschmue** [onnx/tensorflow-onnx#690]
Ops added or changed as of 10/10

Status Legend:
*  \+ = checked into master, without unit test
* +F = checked into master, with unit test
* -N = No change needed
* blank = needs investigation

| Operator | Notes | Prev Opset | Status | Colloborator |
| --- | --- | --- | --- | --- |
| Loop | allow zero variadic inputs: Inputs (3+) -> (2+) | 1 | + |  JP |
| DepthToSpace | added column-row-depth order mode | 1 | +F |  JP |
| Equal | expanded input tensor type to include all numerical types | 7 | +F |  GS  |
| Constant | added sparse tensor support | 9 |  -N | RS |
| Clip | min and max attributes changed to inputs | 6 | +F  |  RS |
| Resize | expanded input tensor type to include all numerical types | 10 | +F |  RS  |
| Gemm | marked input bias tensor C as optional | 9 | -N | RS |
| If | Relax shape inference rule | 1 |  +  |  JP |
| NonMaxSuppression | Add shape inference | 10 | + #707 | JP  |
| ArgMax | Negative axis support | 1 | + |  JP  |
| ArgMin | Negative axis support | 1 | + |  JP |
| Compress | Negative axis support | 9 | -N |   |
| Concat | Negative axis support | 4 | + |  JP |
| Flatten | Negative axis support | 9 | + |  JP  |
| Gather | Negative axis support | 1 | + | JP  |
| Hardmax | Negative axis support | 1 | -N |   |
| LogSoftmax | Negative axis support | 1 | + | JP  |
| OneHot | Negative axis support | 9 | + | JP  |
| ReduceL1 | Negative axis support | 1 | -N |   |
| ReduceL2 | Negative axis support | 1 | -N |   |
| ReduceLogSum | Negative axis support | 1 | -N |   |
| ReduceLogSumExp | Negative axis support | 1 |  -N |   |
| ReduceMax | Negative axis support | 1 | + |  JP |
| ReduceMean | Negative axis support | 1 | + |  JP |
| ReduceMin | Negative axis support | 1 | + | JP  |
| ReduceProd | Negative axis support | 1 | + |  JP |
| ReduceSum | Negative axis support | 1 | + | JP  |
| ReduceSumSquare | Negative axis support | 1 | -N |   |
| Scan | Negative axis support | 9 | + | JP  |
| Slice | Negative axis support | 10 | + #707  | JP  |
| Softmax | Negative axis support | 1 | + |  JP  |
| Split | Negative axis support | 2 | + |  JP |
| Squeeze | Negative axis support | 1 | + |  JP  |
| TopK | "Negative axis support , new attrs" | 10 | + |  JP |
| Unsqueeze | Negative axis support | 1 | -N |   |
| AveragePool | Updated documentation | 10 | + |  JP |
| Conv | Updated documentation | 1 | + |  JP  |
| ConvTranspose | Updated documentation | 1 | +F | RS|
| LpPool | Updated documentation | 2 | -N |   |
| MaxPool | Updated documentation | 10 | + |  JP |
| MaxUnpool | Updated documentation | 9 | -N |   |
| Pad | "Updated documentation,  'pads' attr is now an input" | 2 | +F  | JP  |
| BitShift |  | N/A |  +F |   |
| CumSum |  | N/A |  +F |   |
| Det |  | N/A |  +F |   |
| DynamicQuantizeLinear |  | N/A | -N ? |   |
| GatherElements |  | N/A | -N |   |
| GatherND | graduated from contrib op | N/A | +F  #702 |  JP |
| Range | graduated from contrib op | N/A |  + #707 | JP  |
| Round |  | N/A |  +F |  JP |
| ScatterElements |  | N/A | -N |   |
| ScatterND |  | N/A | +F #702 | JP  |
| Unique |  | N/A |  +F  #707 | JP  |
| ConcatFromSequence |  | N/A | -N |   |
| SequenceAt |  | N/A |  -N |   |
| SequenceConstruct |  | N/A |  -N |   |
| SequenceEmpty |  | N/A |  -N |  |
| SequenceErase |  | N/A | -N | |
| SequenceInsert |  | N/A | -N |  |
| SequenceLength |  | N/A |  -N |   |
| SplitToSequence |  | N/A |  -N |   |
| Scatter |  | deprecated | -N | RS  |

**Comment by jignparm** [comment]
Completed.
